commit b4264c55235bdb036e2994d0fe7fa9619e668862
Author: Mike Hiretsky <mh@calculate.ru>
Date:   Thu Jul 28 12:08:08 2011 +0400

    Fix detect primary,extended,logical partitions type.
    
    Udev info for some partition not contains ID_PART_TABLE_TYPE.
    Get that information from ID_PART_ENTRY_SCHEME.

diff --git a/pym/cl_utils.py b/pym/cl_utils.py
index 23616bc..bea797c 100644
--- a/pym/cl_utils.py
+++ b/pym/cl_utils.py
@@ -1227,7 +1227,7 @@ def getRaidPartitions(raidpath):
 
 def getPartitionType(prop):
     """Get type of dos part table (primary,extended or logical)"""
-    if prop.get('ID_PART_TABLE_TYPE') == 'dos':
+    if prop.get('ID_PART_ENTRY_SCHEME') == 'dos':
         partId = prop.get('ID_PART_ENTRY_TYPE','')
         partNumber = prop.get('ID_PART_ENTRY_NUMBER','')
         if partId and partNumber:
